xml/security_ldap.xml

```diff
@@ -74,41 +74,12 @@
   complex update or query mechanisms. All applications are guaranteed to
   access this service quickly and easily.
  </para>
- <sect1 xml:id="sec.security.ldap.vs_nis">
-  <title>LDAP versus NIS</title>
 
-  <remark condition="generic"> 2007-05-29 - jjaeger: this still necessary? Why not discard the NIS
-   part altogether and merge the use cases for LDAP with the intro? </remark>
-
-  <para>
-   Unix system administrators traditionally use NIS (Network Information
-   Service) for name resolution and data distribution in a network. The
-   configuration data contained in the files <filename>group</filename>,
-   <filename>hosts</filename>, <filename>mail</filename>,
-   <filename>netgroup</filename>, <filename>networks</filename>,
-   <filename>passwd</filename>, <filename>printcap</filename>,
-   <filename>protocols</filename>, <filename>rpc</filename>, and
-   <filename>services</filename> in the <filename>/etc</filename> directory
-   is distributed to clients all over the network. These files can be
-   maintained without major effort because they are simple text files. The
-   handling of larger amounts of data, however, becomes increasingly
-   difficult because of nonexistent structuring.
-   <remark>sknorr, 2014-08-26: "nonexistent structuring" sounds a bit
-    complicated.</remark>
-   NIS is only designed for Unix platforms, and is not suitable as a
-   centralized data administration tool in heterogeneous networks.
-  </para>
 
   <para>
    Unlike NIS, the LDAP service is not restricted to pure Unix networks.
-   Windows&trade; servers (starting with Windows 2000) support LDAP as a
-   directory service. The
-   application tasks mentioned above are additionally supported in non-Unix
-   systems.
-  </para>
-
-  <para>
-   The LDAP principle can be applied to any data structure that needs to be
+   Windows&trade; servers support LDAP as a directory service. The LDAP principle
```

Comment:
This is a bit confusing as a sentence to me?

Reply:
It also was not entirely clear to me what the sentence was referring to, therefore I removed it, including the following lists and the next para.